scripts/download_cams_pm25.py

```
@@ -0,0 +1,267 @@
+        logger.info(f"Data type: {'Forecast' if use_forecast else 'Reanalysis'}")
+        
+        # Create output directory
+        output_dir = output_dir or Path(f"data/cams/processed")
+        output_dir.mkdir(parents=True, exist_ok=True)
+        
+        # Prepare CSV file
+        timestamp_str = datetime.now().strftime('%Y%m%d_%H%M%S')
+        output_file = output_dir / f"cams_pm25_{start_date.strftime('%Y%m%d')}_{end_date.strftime('%Y%m%d')}_{timestamp_str}.csv"
+        
+        # Write header
+        with open(output_file, 'w') as f:
```

Ruff (0.12.2)

Async functions should not open files with blocking methods like `open`

(ASYNC230)